.github/workflows/test-environment.yml

```diff
@@ -10,6 +10,8 @@ on:
         description: |
           Name your environment (Only a-zA-Z0-9 and `-`).
           For example: john-8-7-2-June01'
+          Supported pattern: [a-zA-Z][-a-zA-Z0-9]*
+          Max chars: 20
```

Comment:
can we make this entire section shorter?

Reply:
fixed